### ISSUE TITLE
Install fails: no primary site

### ISSUE THREAD

**Issue by kobeaerts** [craftcms/cms#3777]
### Description

When trying to install using a project.yaml it fails with this error message `An error occurred while executing the "craft\migrations\Install migration: No primary site exists`

### Steps to reproduce

Install from project.yaml file

### Additional info

- Craft version: Craft Solo 3.1.7
- PHP version: 7.2.0
- Database driver & version: MySQL 5.7.20
- Plugins & versions: see screenshot https://i.imgur.com/xsgeuDV.png

This looks like the relevant information in my project.yaml
```
siteGroups:
  34b6169f-19a9-4538-88f0-f10bd01737da:
    name: 'Craft CMS'
sites:
  57bc1f9c-a280-4439-906c-768c2a024287:
    baseUrl: $SITE_URL
    handle: default
    hasUrls: true
    language: en
    name: 'Craft CMS'
    primary: true
    siteGroup: 34b6169f-19a9-4538-88f0-f10bd01737da
    sortOrder: 1
```

.env
```
ENVIRONMENT="development"

SITE_NAME="De Beestige Boerderij"
SITE_URL="http://localhost:8080"

DB_SERVER="localhost"
DB_DATABASE="maxiwebsite-clean"
DB_USER="homestead"
DB_PASSWORD="secret"

ENABLE_WEBSHOP="true"
ENABLE_SEARCH="true"
ENABLE_NEWSLETTER="true"

NEWSLETTER_API_ENDPOINT=""
NEWSLETTER_API_KEY=""
NEWSLETTER_LIST_ID=""

DEFAULT_SITE_URL="http://maxiwebsite.local"
```

**Comment by andris-sevcenko** [comment]
It's more likely that this is caused by a different error. Can you send the entire project.yaml file over to support@craftcms.com, please?

**Comment by kobeaerts** [comment]
You've got mail!

**Comment by andris-sevcenko** [comment]
Fixed via craftcms/google-cloud@4de5630340330e5cbcc6e64fb5383eafd1119d21
